modules/unsupported/mbstyle/src/test/java/org/geotools/mbstyle/transform/StyleTransformTest.java

```diff
@@ -256,6 +256,16 @@ public void testLineGapWidth() throws IOException, ParseException {
 
         // Find the LineMBLayer and assert it contains the correct FeatureTypeStyle.
         MBStyle mbStyle = new MBStyle(jsonObject);
+        StyledLayerDescriptor sld = mbStyle.transform();
+        SLDTransformer styleTransform = new SLDTransformer();
+        String xml;
+		try {
+			xml = styleTransform.transform(sld);
+	        System.out.print(xml);
+		} catch (TransformerException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
```

Comment:
Looks like tabs are being used here - should be spaces.

Reply:
I actually meant to remove that before pushing - I was just using that to debug/test the sld output.